src/Options/Definitions.js

```diff
@@ -898,3 +904,23 @@ module.exports.AuthAdapter = {
     default: true,
   },
 };
+module.exports.LogLevels = {
+  triggerAfter: {
+    env: 'PARSE_SERVER_LOG_LEVELS_TRIGGER_AFTER',
```

Comment:
Just to make sure, was `PARSE_SERVER_LOG_LEVELS_TRIGGER_AFTER` created running `npm run definitions`, or did you manually add this?

Reply:
I used ```npm run definitions```